01-getting-started/002-authenticate.md

```diff
@@ -65,9 +65,10 @@ $ kubectl config set-context gce --user=cluster-admin --namespace=foo \
   && kubectl config use-context gce
 ```
 For more information, please see this handy [cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/).
- 
- ## Where to go from here?
+
+## Where to go from here?
+
  Why don't you try and create a namespace of your own and deploying an application.
- [Create a namespace](https://ministryofjustice.github.io/cloud-platform-user-docs/cloud-platform/env-create/#creating-a-cloud-platform-environment)
- [Deploying an application](https://ministryofjustice.github.io/cloud-platform-user-docs/cloud-platform/app-deploy/#deploying-an-application-to-the-cloud-platform)
 
+ * [Create a namespace](/01-getting-started/003-env-create/)
```

Comment:
As I mentioned on Slack, most of the links that you have starting with `/` won’t work when pushed to GitHub Pages because this site ends up in a sub directory of `ministryofjustice.github.io`. Instead, you need to [pass the URL through a filter](https://jekyllrb.com/docs/github-pages/#project-page-url-structure): `{{ “/01-getting-started/003-env-create/“ | relative_url }}` or use the [`link` Jekyll tag](https://jekyllrb.com/docs/templates/#link): `{% link /01-getting-started/003-env-create.md %}`.

That way, they’ll work both locally and on the published GitHub Pages site.